pkg/services/ngalert/store/alert_rule_test.go

```diff
@@ -371,13 +373,13 @@ func TestIntegration_GetAlertRulesForScheduling(t *testing.T) {
 		{
 			name:    "with populate folders enabled, it returns them",
 			rules:   []string{rule1.Title, rule2.Title},
-			folders: map[string]string{rule1.NamespaceUID: models.GetNamespaceKey(parentFolderUid, rule1.Title), rule2.NamespaceUID: rule2.Title},
+			folders: map[string]string{rule1.NamespaceUID: path.Join(parentFolderTitle, rule1.Title), rule2.NamespaceUID: rule2.Title},
```

Comment:
We could have a title with a backslash in it to ensure that we are escaping it as expected.